LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2020 The Notation Authors
```

Comment:
Should it be 2019-2023 The Notary v2 and Notation Authors

Reply:
+1 to @iamsamirzon proposal

Reply:
To keep this copyright simple and reusable in all Notary repos, I think `2019-2023 The Notary Authors` is appropriate since Notary is the project org/branding name.